tests/backends/database/test_es.py

```diff
@@ -263,11 +265,12 @@ def test_backends_database_es_put_with_badly_formatted_data_raises_a_bulkindexer
     )
 
     # By default, we should raise an error and stop the importation
-    with pytest.raises(BulkIndexError):
+    with pytest.raises(BulkIndexError) as exception_info:
         database.put(sys.stdin, chunk_size=2)
     es.indices.refresh(index=ES_TEST_INDEX)
     hits = es.search(index=ES_TEST_INDEX)["hits"]["hits"]
     assert len(hits) == 5
+    assert exception_info.value.args[-1] == 5
```

Comment:
To be honest, I neither) Good Idea to test that) the expected message matches. However, there is a slight change: the full message becomes: "('1 document(s) failed to index.', 5)" instead of ''1 document(s) failed to index.".